### ISSUE TITLE
[prefer-code-point] doesn't report on `const fromCharCode = String.fromCharCode.bind(String)`

### ISSUE THREAD

**Issue by JounQin** [sindresorhus/eslint-plugin-unicorn#2613]
```js
const fromCharCode = String.fromCharCode.bind(String)
```

**Comment by fisker** [comment]
If someone wants to work on this,  change this part https://github.com/sindresorhus/eslint-plugin-unicorn/blob/756dbbf0a359a139745b092a676f6e535cb85128/rules/prefer-code-point.js#L19-L26 to use `isMemberExpression` check and update related logic should work.

**Comment by harshp0205** [comment]
i want to solve this issue, can you assign it to me?
